test/src/tests-core-sdk.js

```diff
@@ -38,11 +34,10 @@ describe('core SDK', function() {
     it('starts new session', function(done) {
         mParticle.startNewSession();
 
-        var data = getEvent(mockServer.requests, MessageType.SessionStart);
-
-        data.should.be.ok();
+        var sessionStartEvent = findEventFromRequest(window.fetchMock._calls, 'session_start');
```

Comment:
all instances of `mockServer.requests` should be replaced with `window.fetchMock._calls` assuming the eventsv2 endpoint is what was previously being mocked.  Identity and other calls may still be using `mockServer`.